src/migrations/20251740066033_user_info.ts

```diff
@@ -0,0 +1,26 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+
+  await knex.schema.createTable('user_info', table => {
+    table.integer('principal_id').notNullable().primary().references('principals.id');
```

Comment:
It looks like the migration failed, but only on MySQL. I suspect this is just missing `unsigned()` for MySQL to pass. Can be merged after!